datascroller/command_line.py

```diff
@@ -30,7 +30,7 @@ def run_demo():
     scroll(df)
 
 
```

Comment:
I imagine this change will be met with some resistance. Splitting the create parser step into its own function makes testing (slightly) easier as we can test the component parts. However, I don't feel strongly about this change if we want to revert.